woocommerce/woocommerce-android
# issue/3151-order-notification-bug-fix

Closes: #3151 

### Description
This PR updates the logic in Order detail screen to refresh every time a new order notification comes in.

### Testing instructions
- Create an order on a WooCommerce site.
- App received the notification. Don't view the order yet.
- Create another order on the site with different products and customer note.
- App received another notification.
- Tap on one of the new order notifications to view the order details.
- Pull down the notification bar and tap on another new order notification.
- Notice the order detail is opened for the order you clicked.

**This PR is in draft till #4633 can be reviewed and merged.**
